pkg/controller/installation/prometheusRules.go

```diff
@@ -64,12 +64,12 @@ func (r *ReconcileInstallation) newAlertsReconciler(logger *logrus.Entry, instal
 					{
 						Alert: fmt.Sprintf("%sOperatorInstallDelayed", strings.ToUpper(installationName)),
 						Annotations: map[string]string{
-							"sop_url": resources.SopUrlAlertsAndTroubleshooting,
+							"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/OperatorInstallDelayed.asciidoc",
```

Comment:
@pmccarthy not a major concert but would it be too late to put these urls out with the others as variables rather than having the text here and in other locations in the pr?

Reply:
@MStokluska mentioned the same above :) Yeah I'll move these out now